scripts/setup_olserver.sh

```diff
@@ -18,9 +18,9 @@ fi
 
 # apt list --installed
 sudo apt-get update
-sudo apt-get install -y docker.io docker-compose
+sudo apt-get install -y docker.io docker compose
```

Comment:
```suggestion
# Remove any old versions and install newer versions of Docker Engine and Docker Compose.
sudo apt-get remove docker docker-engine docker.io containerd runc
DOCKER_VERSION=5:20.10.7~3-0~ubuntu-focal
sudo apt-get install \
    ca-certificates \
    curl \
    gnupg \
    lsb-release

sudo mkdir -m 0755 -p /etc/apt/keyrings
curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg

echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
    $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null

sudo apt-get update

apt-get install -y \
    docker-ce=$DOCKER_VERSION \
    docker-ce-cli=$DOCKER_VERSION \
    containerd.io \
    docker-buildx-plugin \
    docker-compose-plugin \

docker --version        # 19.03.8
```

Reply:
@gobbledy-gook, @cclauss, I tested this on an Ubuntu 20.04 ("Focal") VM, and these steps worked. However, because this script can install stuff on the host itself, I wanted to flag it for heightened scrutiny.

Reply:
My sense is that we should include a URL link to the Docker instructions instead of repeating their text here.  That way if Docker revises the formula then we are looking a current instructions, not outdated ones.  However, we should ___note in bold___ that in the Internet Archive, we ___require___ an extra step at the end of the installation process.  [`apt-get install apparmor`](https://github.com/internetarchive/openlibrary/pull/7626#discussion_r1137582671) which caused an outage today.  :-(

Reply:
@cclauss, I apologize for being a bit slow, but do you mean we should update the script to just provide a link to the Docker directions rather than having the script execute the commands directly?

Reply:
My mistake.  Let’s keep the script but add a comment that contains the Docker url as a reference if things go wrong.